frontend/src/pages/LoginPage.tsx

```diff
@@ -50,15 +50,14 @@ export default function LoginPage() {
                                onChange={onNameChange} value={username}/>
                     <TextField type="password" id="outlined-password" label="Password" required
                                variant="outlined" onChange={onPasswordChange}  value={password}/>
-                    <Button type="submit" variant="contained" endIcon={<SendIcon/>}>
+                    <Button className="loginButton" type="submit" variant="contained" style={{backgroundColor: "white", color: "orange"}} endIcon={<SendIcon/>}>
```

Comment:
Instead of inline style why not use a second cladd like `loginButton highlighted` ?

Reply:
What exactly would be the benefit of a second class instead of inline styles?

Reply:
Inline styles are slower than classes, as the browser has to calculate them individually each time.